### PR TITLE
Update panic.c

### PR DESCRIPTION
**Pull request by Error-Undefined**
We should really use modern terminology, in one month it's 2020

### CODE REVIEW

**Comment by JamesTheHacker** [comment]
Lame. Next ...

**Comment by PF94** [comment]
That's a obvious yikes for me, This isn't a terminology, but in fact, a meme that will die in a few weeks.

File: kernel/panic.c [hunk 1]
```diff
@@ -212,7 +212,7 @@ void panic(const char *fmt, ...)
 	if (len && buf[len - 1] == '\n')
 		buf[len - 1] = '\0';
 
-	pr_emerg("Kernel panic - not syncing: %s\n", buf);
+	pr_emerg("Vibe check - not syncing: %s\n", buf);
```
Comment (PF94) [inline kernel/panic.c]: Not funny. Move on.

Comment (bazhenovrussia) [reply kernel/panic.c]: What???

Comment (MachineThing) [reply kernel/panic.c]: The kernel has not passed the vibe check

**Pull request closed without merging**